resources/user-agents/browsers/access-netfront/access-netfront-nx-generic.json

```diff
@@ -28,6 +28,11 @@
           "device": "Nintendo 3DS"
         },
         {
+          "match": "Mozilla\/* (New Nintendo 3DS *) NX\/*",
```

Comment:
Please expand this pattern with `AppleWebKit\/* (KHTML* like Gecko)` and add this pattern to the NX 3.0 too.